extensions/amp-analytics/amp-analytics.md

```diff
@@ -62,6 +62,7 @@ when the document is first loaded, and each time an `<a>` tag is clicked:
   - `type` This optional attribute can be specified to use one of the built-in analytics providers. Currently supported values for type are:
     - `googleanalytics`: Adds support for Google Analytics. More details for adding Google Analytics support can be found at [developers.google.com](https://developers.google.com/analytics/devguides/collection/amp-analytics/).
     - `comscore`: Supports comScore Unified Digital Measurement™ pageview analytics. Requires defining *var* `c2` with comScore-provided *c2 id*.
+    - `chartbeat`: Adds support for Chartbeat. More details for adding Chartbeat support can be found at [support.chartbeat.com](http://support.chartbeat.com/docs/).
```

Comment:
having a link to amp-analytics specific documentation would help pubs a lot. If not, at least some page mentions AMP? If you are planning to update the link with AMP related info then this is fine.